--- ceph-20.0.0-2099-gc62cbb7d/src/CMakeLists.txt.orig	2025-06-09 21:24:59.142635736 -0400
+++ ceph-20.0.0-2099-gc62cbb7d/src/CMakeLists.txt	2025-06-09 21:25:28.172137714 -0400
@@ -849,7 +849,7 @@
 if(WITH_LIBCEPHFS)
   set(libcephfs_srcs libcephfs.cc)
   add_library(cephfs ${CEPH_SHARED} ${libcephfs_srcs})
-  target_link_libraries(cephfs PRIVATE client ceph-common
+  target_link_libraries(cephfs PRIVATE client ceph-common common
     ${CRYPTO_LIBS} ${EXTRALIBS})
   if(ENABLE_SHARED)
     set(libcephfs_version 2.0.0)
