diff --git a/CMakeLists.txt b/CMakeLists.txt
index f979951..481deb7 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -27,7 +27,7 @@ endif()
 # Need to search for positioning only after we make sure that it's not WASM.
 # Otherwise we'll get an "QtPositioning not found" error in WASM build.
 find_package(Qt6 ${PROJECT_VERSION} CONFIG OPTIONAL_COMPONENTS
-    Qml Quick Network Test QuickTest Positioning PositioningQuick QuickShapesPrivate
+    Qml Quick Network Test QuickTest Positioning PositioningQuick QuickShapes
 )
 
 find_package(Qt6 ${PROJECT_VERSION} QUIET CONFIG OPTIONAL_COMPONENTS
@@ -43,6 +43,6 @@ macro(assertTargets)
     endforeach()
 endmacro()
 
-assertTargets(Qml Quick Network Test QuickTest Positioning PositioningQuick QuickShapesPrivate)
+assertTargets(Qml Quick Network Test QuickTest Positioning PositioningQuick QuickShapes)
 
 qt_build_repo()
